about.html

```diff
@@ -7,58 +7,80 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
```

Comment:
While not required in this project, I also like to split out my .css into separate files, it is easier to read for me.